src/routes/news/components/Tab.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+
+import React from 'react'
+import styled from 'styled-components'
+import tuNewsLogoActive from '../assets/tu-news-active.png'
+import tuNewsLogoInActive from '../assets/tu-news-inactive.png'
+
+const TU_NEWS = 'tu'
+const decideColor = (activeTab, label, theme) => {
+  let color
+  if (activeTab.toLowerCase() !== 'news') {
+    color = theme.colors.themeColor
+  } else {
+    color = '#0279a6'
```

Comment:
Make color code a constant?

Reply:
Maybe it makes sense to put this into our ThemeProvider and the Theme definition.